### PR TITLE
Fix local env so that developers can import LOIs.

### PR DESCRIPTION
**Pull request by DaoyuT**

### CODE REVIEW

File: functions/src/common/auth.ts [hunk 1]
```diff
@@ -59,16 +60,26 @@ export async function setSessionCookie(req: https.Request, res: Response): Promi
   res.cookie(SESSION_COOKIE_NAME, cookie, { maxAge: expiresIn, httpOnly: true, secure: true });
 }
 
-function getRole(user:DecodedIdToken, survey: DocumentSnapshot): string | null {
-  const acl = survey.get('acl')
+const OWNER_ROLE = 'OWNER';
```
Comment (gino-m) [inline functions/src/common/auth.ts:63]: Consider moving consts to top of file.

**Review by gino-m — CHANGES_REQUESTED** [review]
One nit, LG!

File: functions/src/common/auth.ts [hunk 2]
```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 
-import {DecodedIdToken, getAuth} from 'firebase-admin/auth';
-import {DocumentSnapshot} from 'firebase-admin/firestore';
-import {https, Response} from 'firebase-functions/v1';
+import { DecodedIdToken, getAuth } from 'firebase-admin/auth';
+import { DocumentSnapshot } from 'firebase-admin/firestore';
+import { https, Response } from 'firebase-functions/v1';
+import { EmulatorIdToken } from '../handlers';
```
Comment (gino-m) [inline functions/src/common/auth.ts:20]: You may also need to remove the space between brackets.

File: functions/src/common/auth.ts (hunk 1 above)
Comment (DaoyuT) [reply functions/src/common/auth.ts]: SG, done.

**Review by gino-m — DISMISSED** [review]

File: functions/src/common/auth.ts (hunk 2 above)
Comment (DaoyuT) [reply functions/src/common/auth.ts]: Done.

**Review by gino-m — APPROVED** [review]

**Comment by DaoyuT** [comment]
Fix #1711

**Pull request merged**